app/uk/gov/hmrc/thirdpartyapplication/controllers/ApplicationController.scala

```diff
@@ -82,24 +85,32 @@ class ApplicationController @Inject()(val applicationService: ApplicationService
   val apiGatewayUserAgent: String = "APIPlatformAuthorizer"
 
   def create = requiresAuthenticationFor(PRIVILEGED, ROPC).async(parse.json) { implicit request =>
-    def onV2(createApplicationRequest: CreateApplicationRequest)( fn: CreateApplicationRequestV2 => Future[HasSucceeded]) = 
+    val E = EitherTHelper.make[String]
```

Comment:
We should probably have moved these two lines (88-89) to private class scope (my fault)